src/main/java/com/CezaryZal/api/report/shortened/manager/creator/ShortReportCreator.java

```diff
@@ -0,0 +1,58 @@
+package com.CezaryZal.api.report.shortened.manager.creator;
+
+import com.CezaryZal.api.day.model.ObjectToSaveDay;
+import com.CezaryZal.api.limits.manager.checker.LimitsChecker;
+import com.CezaryZal.api.limits.manager.repo.DailyLimitsRepoService;
+import com.CezaryZal.api.limits.model.LimitsCleanDate;
+import com.CezaryZal.api.meal.manager.repo.MealRepoService;
+import com.CezaryZal.api.report.shortened.manager.repo.ShortReportRepoService;
+import com.CezaryZal.api.report.shortened.model.entity.ShortReport;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class ShortReportCreator {
```

Comment:
Czemu ta logika z tej klasy nie jest w ShortReportService?
Ja bym właśnie tam szukał wszelkich operacji jakie mogę wykonać na obiekcie ShortReport.
Nawet jeśli z jakiegoś powodu chciałbyś oddelegować część kodu do innej klasy, żeby Service nie był jakoś super długi, to ja bym ten ShortReportCreator uznał jako pomocniczy  i wstrzyknał go sobie do ShortReportService i przez niego wystawił metody tego pomocniczego.

Ta uwaga dotyczy wszystkich kreatorów.

Reply:
Ale tak jest.
Do ...Service wstrzykiwana jest klasa pomonicza ...Creator.
Tylko serwisy mają dostęp do creatora obiektu.
Tutaj ewentualnie nazewnictwo nie jest właściwe, bo shortReport odnosi się do Day a nie Report, ale zmienie żeby było bardziej intuicyjne